packages/@react-spectrum/button/src/FieldButton.tsx

```diff
@@ -23,27 +24,32 @@ import {useHover} from '@react-aria/interactions';
 interface FieldButtonProps extends ButtonProps, DOMProps, StyleProps {
   isQuiet?: boolean,
   isActive?: boolean,
-  validationState?: 'valid' | 'invalid'
+  validationState?: 'valid' | 'invalid',
+  focusRingStyles?: CSSModule
 }
 
 // @private
 function FieldButton(props: FieldButtonProps, ref: FocusableRef) {
+  props = useSlotProps(props, 'button');
```

Comment:
Thoughts on this approach for propagating focus ring overrides?

Reply:
The extent of my logic was that they should share the same slot name since they are both buttons and may end up being used in similar slot positions lol. Happy to change it to `fieldbutton` or something

Reply:
yeah, i had that thought as well, i haven't settled on a strong opinion yet, lets see if @devongovett has a strong opinion